packages/manager/apps/dedicated/client/app/account/user/user.html

```diff
@@ -1,5 +1,8 @@
 <div class="user-account">
-    <oui-header data-heading="{{ :: 'user_account_title_infos' | translate }}">
+    <oui-header
+        data-heading="{{ :: 'user_account_title_infos' | translate }}"
+        data-description="{{ :: $ctrl.coreConfig.getRegion() !== 'US' ? 'user_account_description' : '' | translate }}"
```

Comment:
Could this condition been defined as a feature flipping rule? It is used 4 times

Reply:
@JeremyDec  In the future there will probably also be a description for US but different from EU and CA (as not the same functions are available).

Reply:
Question: What is the description shown for US region?

Reply:
It is planned in the future to have US content, but it is not yet adapted and ready.